docs/posts/roadmap-2024-H1/index.qmd

```diff
@@ -0,0 +1,383 @@
+---
+title: "Ibis project 2024 roadmap"
+author: "Cody"
+date: "2024-02-05"
+thumbnail: commits.png
+categories:
+    - blog
+    - roadmap
+    - community
+---
+
+## Overview
+
+The Ibis project is an [independently governed open-source
+community](https://github.com/ibis-project/governance) that builds the portable
+Python dataframe library. It is primarily backed by [Voltron
+Data](https://voltrondata.com) but has
+[contributors](https://github.com/ibis-project/ibis/graphs/contributors) across
+a range of data companies and institutions.
+
+This is the first public roadmap for the Ibis project. Thus, there's a bit of
+backstory to explain how we got here and setup the context for where we're
+going. Feel free to [skip ahead to the roadmap](#2024-roadmap) if you're already
+familiar with Ibis.
+
+## Background
+
+The Ibis project was started in 2015 by [Wes McKinney](https://wesmckinney.com),
+the creator of pandas, as a pandas-like interface to Apache Impala. It received
+improvements and support over the years, but really took off under the
+stewardship of [Phillip Cloud](https://github.com/cpcloud) and the [current Ibis
+team at Voltron Data](#who-are-the-core-contributors). It now supports 20+
+backends (including Apache Impala!) and is improving rapidly. It's never been a
+better time to get involved with Ibis.
+
+You can see the inflection point in the number of commits to the repository in
+early 2022:
+
+![Ibis commits over time](commits.png)
+
+### Who am I?
+
+My name is Cody and I'm employed by Voltron Data to work on Ibis full-time as a
+Technical Product Manager. I am an Ibis committer and have contributed Delta
+Lake table input/output methods, helped moved the documentation over to
+[Quarto](https://quarto.org), and created the [Zulip
+chat](https://ibis-project.zulipchat.com) for the community.
+
+My job is to help the Ibis community grow and thrive. I have a background in ML
+(especially MLOps) and data products. Ibis solves many challenges I've seen in
+the data space and I'm excited to help increase its adoption as a standard
+Python frontend for dozens of data backends to reduce friction in the data
+ecosystem.
+
+### Why does Voltron Data (VoDa) support Ibis?
+
+Why does Voltron Data employ a Technical Product Manager to work on Ibis
+full-time? Why does Voltron Data employ five software engineers to work on Ibis
```

Comment:
Do we need the number "five" in here, given that some folks might be joining the team in the future?

Reply:
I'll add "currently" or something, but blogs are point in time. I was going to say "(and counting!)" or something but also didn't want to commit to that publicly